templates/extra/terms.html

```diff
@@ -1,57 +1,57 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Terms of Use</h1>
+    <h2>Terms of Use</h2>
     <div class="break">
         <hr>
     </div>
-    <div class="extra-padding">
+    <div class="extra-padding-wide">
         <h5>Last updated: October 12, 2015</h5>
-        <p>Please read these Terms of Service ("Terms", "Terms of Service", "Terms and Conditions") carefully before using the http://www.investperdiem.com website (the "Service") operated by PerDiem, LLC ("us", "we", or "our").</p>
-        <p>Your access to and use of the Service is conditioned upon your acceptance of and compliance with these Terms. These Terms apply to all visitors, users and others who wish to access or use the Service.</p>
-        <p>By accessing or using the Service you agree to be bound by these Terms. If you disagree with any part of the terms then you do not have permission to access the Service.</p>
+        <p class="left-align">Please read these Terms of Service ("Terms", "Terms of Service", "Terms and Conditions") carefully before using the http://www.investperdiem.com website (the "Service") operated by PerDiem, LLC ("us", "we", or "our").</p>
```

Comment:
Do not apply `class="left-align"` to every `<p>` in `terms.html`. Do what you did in `faq.html`.